integration-tests/ci-visibility.spec.js

```diff
@@ -404,16 +400,275 @@ tests.forEach(({
           runTestsWithCoverageCommand,
           {
             cwd,
-            env: {
-              ...process.env,
-              DD_API_KEY: '1',
-              DD_APP_KEY: '1',
-              DD_CIVISIBILITY_AGENTLESS_ENABLED: 1,
-              DD_CIVISIBILITY_AGENTLESS_URL: `http://127.0.0.1:${receiver.port}`,
-              DD_CIVISIBILITY_GIT_UPLOAD_ENABLED: 1,
-              DD_CIVISIBILITY_ITR_ENABLED: 1,
-              NODE_OPTIONS: '-r dd-trace/ci/init'
-            },
+            env: getAgentlessConfig(receiver.port),
+            stdio: 'inherit'
+          }
+        )
+      })
+    })
+
+    describe('evp proxy', () => {
+      it('does not do any ITR request if the agent is not EVP compatible', (done) => {
```

Comment:
Can we please avoid all these abbreviations? Makes it hard for others to contribute when we're using all these unfamiliar terms.

Reply:
good point. Changing it

Reply:
fixed in https://github.com/DataDog/dd-trace-js/pull/2584/commits/b7ffb778d8e282067ea59ba73348099523d68d9c